clients/web/src/templates/widgets/itemList.jade

```diff
@@ -3,9 +3,15 @@ ul.g-item-list
     li.g-item-list-entry
       if checkboxes
         input.g-list-checkbox(type="checkbox", g-item-cid="#{item.cid}")
-      a.g-item-list-link(g-item-cid="#{item.cid}")
+      a.g-item-list-link.g-right-border(g-item-cid="#{item.cid}")
         i.icon-doc-text-inv
         | #{item.get('name')}
+      a(title="Download item" href=item.downloadUrl())
+        i.icon-download
+      if item.collection.length === 1
```

Comment:
This doesn't work. `item.collection` is the collection of items that this item belongs to, not its children files.